plugins/router/f5/plugin_test.go

```diff
@@ -626,6 +637,60 @@ func patchVserverHandler(f5state mockF5State) http.HandlerFunc {
 	}
 }
 
+func validatePartitionPath(response http.ResponseWriter, request *http.Request,
+	f5state mockF5State, partitionPath string) bool {
+	_, ok := f5state.partitionPaths[partitionPath]
+	if !ok {
+		response.WriteHeader(http.StatusNotFound)
+		fmt.Fprintf(response,
+			`{"code":404,"errorStack":[],"message":"01020036:3: The requested folder (%s) was not found."}`,
+			partitionPath)
+		return false
+	}
+
+	return true
+}
+
+func getPartitionPath(f5state mockF5State) http.HandlerFunc {
+	return func(response http.ResponseWriter, request *http.Request) {
+		vars := mux.Vars(request)
+		partitionPath := vars["partitionPath"]
+
+		if !validatePartitionPath(response, request, f5state, partitionPath) {
+			return
+		}
+
+		fullPath := strings.Replace(partitionPath, "~", "/", -1)
```

Comment:
I kind of think this should be `fullPath := f5state.partitionPaths[partitionPath]` to get the verbatim path that was POSTed and stored by `postPartitionPathHandler`.  Is there any reason not to do so?